### DISCUSSION TITLE
Codespaces - infinite loop of "Attempting to reconnect" no option to view container logs

### DISCUSSION THREAD

**Discussion by altjx** [community/community#75824] [Codespaces]
### Select Topic Area

Question

### Body

I'm trying to troubleshoot what's going on with my GitHub Codespace -- I'm not sure if it's an issue in the Dockerfile or if it's the `devcontainer.json` file. However, I am unable to view any container logs because it's not presenting the option to me to view the container logs. Instead, it's just giving me a constant "Attempting to reconnect" message.

Here's what I see when I go to Code -> Start Codespace:

![Screenshot 2023-11-16 at 5 11 59 PM](https://github.com/community/community/assets/4023423/26f42640-87ea-4bd4-a9bf-3077fdfd503c)

Reloading the window quickly flashes to a "Workspace does not exist" screen, followed by VS code with the repository's contents in it (as normal). However, I can clearly tell it didn't build the container, so it must be in recovery mode.

Is there a way to view the container logs manually when it's not prompting me automatically?

**Comment by altjx** [comment]
Seems like I was able to go to "Codespaces" in the command pallet to view the creation logs, so I'm good on this one.

**Comment by Tusharssinghh** [comment]
I'm having the same problem, please tell me how to resolve this issue ?
